specifyweb/frontend/js_src/lib/components/SchemaViewer/TableList.tsx

```diff
@@ -100,14 +144,13 @@ export function SchemaViewerTableList<
 
 function Cell({
   children,
+  className = '',
```

Comment:
this component is useless now that it doesn't do much at all - consider inlining it
(it adds more code keeping this component separate than inlining it in the component above lol)